tmt/cli.py

```diff
@@ -263,6 +262,15 @@ def provision(context, **kwargs):
 @click.option(
     '-p', '--playbook', metavar='PLAYBOOK',
     help='Path or URI to ansible playbook to run.')
+@click.option(
+    '-g', '--package', metavar='PACKAGE',
```

Comment:
Let's skip the short option for now. Once the step is converted into a dynamic plugin we can add a `-p` option insteady which would be more intuitive.

Reply:
OK.